client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -0,0 +1,230 @@
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import GSuiteNewUserList from 'calypso/components/gsuite/gsuite-new-user-list';
+import { hasDiscount } from 'calypso/components/gsuite/gsuite-price';
+import InfoPopover from 'calypso/components/info-popover';
+import PromoCardPrice from 'calypso/components/promo-section/promo-card/price';
+import { getSelectedDomain } from 'calypso/lib/domains';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import { getAnnualPrice, getGoogleMailServiceFamily, getMonthlyPrice } from 'calypso/lib/gsuite';
+import { newUsers } from 'calypso/lib/gsuite/new-users';
+import withCartKey from 'calypso/my-sites/checkout/with-cart-key';
+import EmailProvidersStackedCard from 'calypso/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card';
+import {
+	EmailProvidersStackedCardProps,
+	ProviderCard,
+} from 'calypso/my-sites/email/email-providers-stacked-comparison/provider-cards/provider-card-props';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getProductBySlug, getProductsList } from 'calypso/state/products-list/selectors';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+import './google-workspace-card.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+const identityMap = ( item: any ) => item;
+
+const getGoogleFeatures = () => {
+	return [
+		translate( 'Annual billing' ),
+		translate( 'Send and receive from your custom domain' ),
+		translate( '30GB storage' ),
+		translate( 'Email, calendars, and contacts' ),
+		translate( 'Video calls, docs, spreadsheets, and more' ),
+		translate( 'Work from anywhere on any device – even offline' ),
+	];
+};
+
+const googleWorkspaceCardInformation: ProviderCard = {
+	detailsExpanded: true,
+	expandButtonLabel: translate( 'Expand' ),
+	onExpandedChange: noop,
+	providerKey: 'google',
+	showExpandButton: true,
+	description: translate(
+		'Professional email integrated with Google Meet and other productivity tools from Google.'
+	),
+	logo: { path: googleWorkspaceIcon, className: 'google-workspace-icon' },
+	productName: getGoogleMailServiceFamily(),
+	features: getGoogleFeatures(),
+};
+
+const GoogleWorkspaceCard: FunctionComponent< EmailProvidersStackedCardProps > = ( props ) => {
+	const { currencyCode = '', domain, gSuiteProduct, selectedDomainName } = props;
+	const googleWorkspace: ProviderCard = googleWorkspaceCardInformation;
+
+	const isUpgrading = () => {
+		const { domain, hasCartDomain } = props;
+
+		return ! hasCartDomain && hasEmailForwards( domain );
+	};
+
+	const productIsDiscounted = hasDiscount( gSuiteProduct );
+
+	const monthlyPrice = translate( '{{price/}} /mailbox', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{price/}} /mailbox /month' )` *ES Score: 8*